modules/storage-persistent-storage-nfs-selinux.adoc

```diff
@@ -5,20 +5,22 @@
 [id="nfs-selinux_{context}"]
 = SELinux
 
-By default, SELinux does not allow writing from a Pod to a remote
-NFS server. The NFS volume mounts correctly, but is read-only.
+By default, SELinux does not allow writing from a pod to a remote NFS server. The NFS volume mounts correctly but it is read-only.
 
-To enable writing to a remote NFS server, follow the below procedure.
+To enable writing to a remote NFS server, use the following procedure.
+
+[NOTE]
+====
+This procedure is only required for {system-base-full} worker nodes. Functionality is enabled by default in {op-system-first} nodes.
```

Comment:
```suggestion
This procedure is required for only {system-base-full} worker nodes. Functionality is enabled by default in {op-system-first} nodes.
```

Reply:
I see your point about pairing "only" more closely with what it describes but it sounds a bit odd that way to me, so I moved it to the end of the sentence.

Reply:
Sounds good to me!